app/nexus/internal/route/acl/policy/create.go

```diff
@@ -79,34 +78,16 @@ func RoutePutPolicy(
 		return errors.ErrReadFailure
 	}
 
-	// TODO: sanitize
+	err := guardPutPolicyRequest(*request, w, r)
```

Comment:
Poor man's aspect-oriented programming.

input validation and policy enforcement is a cross-cutting concern, so we are abstracting them in functions.